detector/gpg.go

```diff
@@ -12,17 +12,17 @@ import (
 )
 
 // WatchGPG watches for hints that YubiKey is maybe waiting for a touch on a GPG request
-func WatchGPG(gpgPubringPath string, requestGPGCheck chan bool) {
-	// No need for a buffered channel,
-	// we are interested only in the first event, it's ok to skip all subsequent ones
-	events := make(chan notify.EventInfo)
+func WatchGPG(filesToWatch []string, requestGPGCheck chan bool) {
+	events := make(chan notify.EventInfo, len(filesToWatch))
 
 	initWatcher := func() {
-		if err := notify.Watch(gpgPubringPath, events, notify.InOpen, notify.InDeleteSelf, notify.InMoveSelf); err != nil {
```

Comment:
`notify.InDeleteSelf` and `notify.InMoveSelf` here were meant to have resiliency, I don't quite remember the original scenario, but maybe something like if `~/.gnupg` folder was moved, then the app should just re-start gpg detection. Without these, the `default` case below with `GPG received file event '%+v', recreating the watcher.` won't ever be triggered.

Did you remove these on purpose? If not, let's readd them

Reply:
No, I just reviewed the Notify documentation, and I initially thought the InOpen documentation was all we needed. I hadn’t considered resiliency—good point! We'll stick with the old one.